### PR TITLE
Leaderboard: Setup project

### PR DESCRIPTION
**Pull request by GhostEsso**
- I used callbacks and promises.
- I learned how to use proper ES6 syntax.
- I used ES6 modules to write modular JavaScript.
- I used webpack to bundle JavaScript.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,153 @@
...
+- **Add a task to complete**
+- **Check out if the task has been completed**
+- **Remove all completed tasks**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://)
+
```
Comment (omar-muhamad) [inline README.md]: - [ ] If you don't have live demo link of your app yet, Please remove the entire section from the readme file and add it again when you have a working link.

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi @GhostEsso, 

#### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

#### Your Code Review Request is marked as invalid for this reason:

- [x] Please check the inline comments below.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: README.md [hunk 2]
```diff
@@ -1 +1,147 @@
...
+### Deployment
+
+You can deploy this project using:
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
```
Comment (algerina) [inline README.md]: - [x] I recommend deleting this line, your Readme will look more professional.

File: README.md [hunk 3]
```diff
@@ -1 +1,147 @@
...
+
+  - It's a project creative commons, you can use if you give back the credit to the developers
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (algerina) [inline README.md]: - [x] It is preferable to get rid of these lines too.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @GhostEsso ,

Great job so far! 

May I highlight the following: 
- [x] Your project is setup correctly👏  ✔️
- [x] Your pull request is descriptive👏  ✔️
- [x] You used well Gitflow👏  ✔️




## CHANGES REQUIRED:muscle::clap:
Check the comments under the review.



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Review by muneebulrehman — APPROVED** [review]
Hi @GhostEsso , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Descriptive PR
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by GhostEsso** [comment]
@muneebulrehman @algerina @Omar-Muhamad Thank you for reviewing my PR

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    const tr = document.createElement('tr');
+    const td1 = this.createTableHeader('Libellé: 100'); // Utilisation correcte de 'this'
+    const td2 = this.createTableHeader('Libellé: 50'); // Utilisation correcte de 'this'
+    const td3 = this.createTableHeader('Libellé: 20'); // Utilisation correcte de 'this'
+    tr.appendChild(td1);
+    tr.appendChild(td2);
+    tr.appendChild(td3);
+    table.appendChild(tr);
+    return table;
+  }
```
Comment (muneebulrehman) [inline src/index.js:80]: - Optional
I would recommend following the wireframe design. You can customize it however you want, but the basic layout should be the same. Also, the refresh button is missing in your case.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

Comment (GhostEsso) [reply src/index.js:80]: @muneebulrehman Okay, I'm going to change it :pray:

Comment (GhostEsso) [reply src/index.js:80]: @muneebulrehman  I did it : 
image

**Pull request merged**